packages/components/psammead-episode-list/src/fixtures.jsx

```diff
@@ -268,10 +268,13 @@ export const renderVideoEpisodes = ({
               </VisuallyHiddenText>
             </EpisodeList.Link>
             {episode.episodeTitle && (
-              <div>
-                <EpisodeList.Metadata as="time" dateTime={episode.dateTime}>
+              <div style={{ display: 'inline' }}>
```

Comment:
The <div> is added so that with no-css, this element breaks onto another line.
The `inline` style is applied so with _with_ css, the element is rendered inline.

Reply:
Hey @jroebu14 - changed this to a styled comp